environments/autostart.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Autostart"
+description: Learn how to configure automated environment rebuilds.
+---
+
+Coder automatically [turns off](../admin/environment-management/shutdown.md)
+unused environments to help you save on cloud usage costs. Turning off
+environments, however, requires you to turn them on manually the next day.
+Autostart can help eliminate this manual process.
```

Comment:
I think it would be nice to have the link span more text:

```md
Coder
[automatically turns off unused environments](../admin/environment-management/shutdown.md)
to help you save on cloud usage costs. Turning off environments, however,
requires you to turn them on manually the next day. Autostart can help eliminate
this manual process.
```